### PR TITLE
i18n(it): Update pages.mdx & plugins.mdx

### PR DESCRIPTION
**Pull request by casungo**
#### Description

Updated `pages.mdx` & `plugins.mdx` italian pages (back to 100% :P)

### CODE REVIEW

File: docs/src/content/docs/it/guides/pages.mdx [hunk 1]
```diff
@@ -104,15 +104,37 @@ Le seguenti proprietà differiscono dal frontmatter di Markdown:
...
+<StarlightPage
+	frontmatter={{ title: 'Orione' }}
+	sidebar={[
+		{ label: 'Home', href: '/' },
+		{
+			label: 'Costellazioni',
+			items: [
+				{ label: 'Andromeda', href: '/andromeda/' },
+				{ label: 'Orione', href: '/orion/', isCurrent: true },
+				{ label: 'Orsa Minore', href: '/ursa-minor/', badge: 'Stub' },
```
Comment (delucis) [inline docs/src/content/docs/it/guides/pages.mdx:130]: I guess we should also translate the badge ([source for this translation](https://it.wikipedia.org/wiki/Aiuto:Abbozzo)):

```suggestion
				{ label: 'Orsa Minore', href: '/ursa-minor/', badge: 'Abbozzo' },
```

**Review by delucis — COMMENTED** [review]
Looks great — thank you @casungo! Just left one suggestion.

File: docs/src/content/docs/it/guides/pages.mdx (hunk 1 above)
Comment (casungo) [reply docs/src/content/docs/it/guides/pages.mdx]: totally right :)

**Review by delucis — APPROVED** [review]
Thanks! Back to 100% 🚀

**Pull request merged**